outputs/core_pools.json

```diff
@@ -88,5 +93,9 @@
     "0x0c659734f1eef9c63b7ebdf78a164cdd745586db000000000000000000000046": "USDC/USDbC/axlUSDC"
   },
   "mode": {},
-  "fraxtal": {}
+  "fraxtal": {
+    "0x1570315476480fa80cec1fff07a20c1df1adfd53000200000000000000000009": "sFRAX/sfrxETH",
+    "0x33251abecb0364df98a27a8d5d7b5ccddc774c42000000000000000000000008": "sFRAX/sDAI/sUSDe",
+    "0xa0b92b33beafce388ce0092afdcd0ca77323eb12000000000000000000000006": "sFRAX/sDAI"
+  }
```

Comment:
@Tritium-VLK did you say you wanted to double check something here?

Reply:
I said we are not sweeping these yet/not running core pools fees yet.  When we fire that up we should do a very detailed check on the first run, at which point I'll go through this list.  For now, it's good to be generating it but it is not in use.